src/backend/mi2/mi2lldb.ts

```diff
@@ -10,7 +9,7 @@ export class MI2_LLDB extends MI2 {
 		// Since the CWD is expected to be an absolute path in the debugger's environment, we can test
 		// that to determine the path type used by the debugger and use the result of that test to
 		// select the correct API to check whether the target path is an absolute path.
-		const debuggerPath = path.posix.isAbsolute(cwd) ? path.posix : path.win32;
+		const debuggerPath = posix.isAbsolute(cwd) ? posix : win32;
```

Comment:
this looks very weird, I don't think the import separation gains us anything here. In node.js `path` used to be a library that you also always just imported as a whole, although I haven't kept up with the JS world there what is used now.

But considering that we are assigning it to `debuggerPath`, so it can be used like `debuggerPath.join(...)` instead of `path.join(...)` to get the correct `path` for the current platform, I'd much prefer keeping the `path.` prefix

Reply:
I reverted all changes to imports.